package-lock.json

```diff
@@ -22975,6 +22977,39 @@
         "url": "https://www.patreon.com/infusion"
       }
     },
+    "node_modules/framer-motion": {
+      "version": "11.18.2",
```

Comment:
I had to manually delete the package-lock's `framer-motion` version for this to pick up. it just gets confused and unable to resolve the deps, resulting in no change.

we've got some problems with our package dependencies right now. I've got a separate PR to deal with that that should unblock this properly if we're not comfortable with this approach.

while inadvisable, the package lock is generated on `npm i` so this should be in the correct state now and you'd never know I manually edited it if I hadn't said so